Source/Other/Extensions/NSImageExtensions.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  NSImageExtensions.swift
+//  Sequel Ace
+//
+//  Created by Jakub Kašpar on 23.12.2020.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+import Foundation
```

Comment:
What's this?

Reply:
Ahh it's used in the second PR for dark mode fixes